test_zonos.py

```
@@ -0,0 +1,572 @@
+
+    # 4. Memory (if enabled)
+    try:
+        mem_table = ka.table(sort_by="self_cpu_memory_usage", row_limit=topk)
+        dump(mem_table, "top_cpu_mem.txt")
+        
+        # CUDA Memory table
+        try:
+            cuda_mem_table = ka.table(sort_by="self_cuda_memory_usage", row_limit=topk)
+            dump(cuda_mem_table, "top_cuda_mem.txt")
+        except Exception:
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)